packages/webapp/src/components/Drawer/style.module.scss

```diff
@@ -67,6 +67,7 @@
 
 .sideDrawer {
   top: var(--global-navbar-height);
+  max-height: calc(100vh - var(--global-navbar-height));
```

Comment:
I missed this on the previous PR, but this is used in conjunction with 100% content (on Drawer + on the Form it contains) to make sure that the Layout footer (= the form buttons) end up on the bottom of the screen, rather than slightly below it!

Reply:
Hmmm, so you mean you would expect the buttons fixed to the bottom of the drawer the way the header of the drawer is fixed to the top? I think it could definitely be done, but the refactor would be pretty significant because all the form is set up in that inner component. How important do you think that distinction is? Should we check with Loïc tomorrow morning?

Reply:
No, I think I expect the scroll bar to extend the full height of the drawer and the bottom white bar not to exist. 

This is not a blocker for me at all -- if you think its not a problem please ignore me haha